src/AccountInfo.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  AccountAddress, AccountAddressInput, AnyPublicKey,
+  Deserializer,
+  Ed25519PublicKey,
+  MultiEd25519PublicKey, MultiKey,
+  PublicKey,
+  Serializable,
+  Serializer, SigningScheme
+} from '@aptos-labs/ts-sdk'
+
+export interface AccountInfoInput {
+  address: AccountAddressInput
+  publicKey: PublicKey
+  ansName?: string
+}
+
+export class AccountInfo extends Serializable {
```

Comment:
what does the wallet return the dapp here? the `AccountInfo` class? I really wish we could have the de/serialization logic live in our sdk

Reply:
yes, we can return an AccountInfo instance

Reply:
do you think this class should live in the sdk? why?

Reply:
dont know if this class but the de\serialization logic

Reply:
what are you proposing?